about.html

```diff
@@ -7,33 +7,75 @@
   <title>Sprint Challenge - About</title>
 
   <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
-  <link rel="stylesheet" href="css/index.css">
+  <link rel="stylesheet" href="index.html">
 
 </head>
+<div class="container">
+  <header>
+    <section>
+   
+      <div class="header">
+
+        <div class="lamda">
+          <div img="symbol" alt="lamda symbol"></div>
+          <div class="lamdatext"><Span>amda</Span></div>
+       </div>
+    </div>
+</div>
+  
+      <div class="nav">
+         <nav>
```

Comment:
If you want a different styling for each anchor then adding a class to it is fine.  But in this case just targeting nav and nav a will get you where you want to be with the nav bar.